app/controllers/assessments_controller.rb

```diff
@@ -2000,78 +1970,6 @@ def get_assessment
     end
   end
 
-  def extend_config_module
```

Comment:
I'm all for making this extend_config_module DRY, but I'm not sure if it's appropriate to put it in autograde.rb

What it essentially does is to let instructors overwrite assessment configs and some of the methods that get overwritten are not really related to autograding. (e.g. there are methods for overwriting scoreboard headers) I guess this works for now since only submissions_controller and assessments_controller uses this method and autograde.rb is included in both, but I'd suggest moving it into something like application helpers. What do you think?